MAUI/Pull-to-Refresh/migration.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Migrate from Xamarin SfPullToRefresh to .NET MAUI PullToRefresh | Syncfusion 
+description: Learn here all about Migrating from Syncfusion Xamarin PullToRefresh to Syncfusion .NET MAUI PullToRefresh control and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfPullToRefresh to .NET MAUI PullToRefresh
+
+To migrate easier from Xamarin.Forms SfPullToRefresh to .NET MAUI PullToRefresh, we kept most of the APIs from Xamarin SfPullToRefresh in MAUI PullToRefresh. However, to maintain the consistency of API naming in .NET MAUI PullToRefresh, we renamed some of the APIs. The APIs that have been changed in MAUI PullToRefresh from Xamarin SfPullToRefresh are detailed as follows.
```

Comment:
do not mention "MAUI" alone.

say ".NET MAUI SfPullToRefresh" 

![image](https://github.com/syncfusion-content/maui-docs/assets/44426156/712f592c-ef73-4aa2-9865-17680ce5d731)